### ISSUE TITLE
Image refresh for fedora-31

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/bots#44]
Image refresh for fedora-31
 * [ ] FAIL: image-refresh fedora-31

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-32946.
Log: https://logs-https-cockpit.apps.ci.centos.org/logs/image-refresh-44-20190930-005454/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-04.
Log: https://209.132.184.41:8493/logs/image-refresh-44-20190930-005454/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-44-20190930-005454/

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-03.
Log: https://209.132.184.41:8493/logs/image-refresh-44-20190930-043535/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-44-20190930-043535/

**Comment by marusak** [comment]
Failed two times with:
```
+ passwd --stdin root
+ echo foobar
Changing password for user root.
passwd: Have exhausted maximum number of retries for service
image-create: setup failed with code 1
```

**Comment by martinpitt** [comment]
Fixed in PR #57
